qiime2/core/archive/provenance.py

```diff
@@ -453,15 +456,21 @@ def make_transformers_section(self):
 
     def make_env_section(self):
         env = collections.OrderedDict()
-        env['platform'] = pkg_resources.get_build_platform()
+        env['platform'] = platform.platform()
         # There is a trailing whitespace in sys.version, strip so that YAML can
         # use literal formatting.
         env['python'] = LiteralString('\n'.join(line.strip() for line in
                                       sys.version.split('\n')))
         env['framework'] = self.make_software_entry(
             qiime2.__version__, qiime2.__website__, self._framework_citations)
         env['plugins'] = self.plugins
-        env['python-packages'] = self.capture_env()
+
+        # sort pkgs alphabetically, ignoring upper/lower casing
+        unsorted_packages = self.capture_env()
```

Comment:
@ebolyen this could probably also go under capture_env() - I don't have strong feelings. But this actually does a better job of sorting than what we were doing before (since it ignores casing).